src/amo/components/ShowMoreCard/index.js

```diff
@@ -23,20 +23,48 @@ type UIStateType = {|
   readMoreExpanded: boolean,
 |};
 
+type TruncateToMaxHeightProps = {|
+  contents: HTMLElement | null,
+  maxHeight: number,
+  setUIState: ($Shape<UIStateType>) => void,
+  uiState: UIStateType,
+|};
+
+export const truncateToMaxHeight: (TruncateToMaxHeightProps) => void = ({
+  contents,
+  maxHeight,
+  setUIState,
+  uiState,
+}: TruncateToMaxHeightProps) => {
```

Comment:
Why do we have to return all the same parameters that were passed in?  (and so necessitating a weird `TruncateToMaxHeightProps` type.)

Reply:
We don't. It's just that the Flow definitions are a bit hard to follow. I realized we don't need to define the type of the function itself, which removes some of the code above and hopefully makes it clearer. It also means we don't need the extra  `TruncateToMaxHeightProps` type.